docs/index.asciidoc

```diff
@@ -33,6 +33,21 @@ type of event can be enriched and transformed with a broad array of input, filte
 native codecs further simplifying the ingestion process. Logstash accelerates your insights by harnessing a greater
 volume and variety of data.
 
+
+[serverless]
+.Logstash to {serverless-full}
+****
+If you are sending data from {ls} to {serverless-full}, note these differences from {ess} and self-managed Elasticsearch:
+
+* {serverless-full} uses {dlm} ({dlm-init}) instead of {ilm} ({ilm-init}). 
+Any {ilm-init} settings in your <<plugins-outputs-elasticsearch,{es} output plugin>> configuration are ignored and may cause errors.
+* Use API keys to access {serverless-full} from {ls}. 
+Any user-based security settings in your in your <<plugins-outputs-elasticsearch,{es} output plugin>> configuration are ignored and may cause errors.
```

Comment:
```suggestion
* Use API keys to access {serverless-full} from {ls}. 
Any user-based security settings in your in your <<plugins-outputs-elasticsearch,{es} output plugin>> configuration are ignored and may cause errors.
* {serverless-full} uses {dlm} ({dlm-init}) instead of {ilm} ({ilm-init}). 
Any {ilm-init} settings in your <<plugins-outputs-elasticsearch,{es} output plugin>> configuration are ignored and may cause errors.
```